README.md

```diff
@@ -1,22 +1,52 @@
-# Shield Study Embedded Web Extension Template
+# Shield Studies Add-On Template
 
 ![CircleCI badge](https://img.shields.io/circleci/project/github/mozilla/shield-studies-addon-template/master.svg?label=CircleCI)
+[![Coverage Status](https://coveralls.io/repos/github/mozilla/shield-studies-addon-template/badge.svg)](https://coveralls.io/github/mozilla/shield-studies-addon-template)
 
 ## Important notice
 
-### We are moving to Web Extensions
+### We are moving to WebExtensions
 
-In an effort to remove the necessity of creating legacy add-ons for Shield studies, we are working on [supporting a pure Web Extension workflow in this template](https://github.com/mozilla/shield-studies-addon-template/issues/53).
-Until support for those workflows become stable, it is still recommended to use this template as it is and create legacy add-ons for your study.
-Chat with us: #shield on Slack about the latest progress and how to help us progress faster away from legacy add-ons.
+This is a work in progress branch for [supporting a pure WebExtension workflow in this template](https://github.com/mozilla/shield-studies-addon-template/issues/53).
+
+The latest template depends on version 5 of [shield-studies-addon-utils](https://github.com/mozilla/shield-studies-addon-utils/) (also work in progress), so until it has been published via npm, you will need to check it out manually and use npm link:
```

Comment:
This is odd. I thought you could do <kbd>$ npm i mozilla/shield-studies-addon-utils#develop -D</kbd> and pull from GitHub directly if it wasn't published to npm yet, but I'm getting some "prebuild" script error where it's trying to find the **npm-run-all** module, which isn't installed in my local dummy folder (and seemingly not getting installed via npm here because it is a devDependency in **shield-studies-addon-utils**).
Interesting that running `npm install` calls a `build` script (and the `prebuild` and presumably `postbuild` hooks as well). Looks like I need to read up on npm event lifecycles again. But I can't find anything informative in https://docs.npmjs.com/cli/install or https://docs.npmjs.com/misc/scripts.

Reply:
Hm... I didn't even know you could do that for unpublished packages. I will have to look into that more.

I was able to test this branch in a different way: I just did `npm run build` in the `develop` branch of SSAU to generate the bundled `api.js` for the `study` API and copied the two folders below into `shield-studies-addon-template/src/privileged/`:
* `shield-studies-addon-utils/webExtensionApis/prefs/`
* `shield-studies-addon-utils/webExtensionApis/study/` (just `schema.json` and `api.js`)

Reply:
@pdehaan I am not up to date with npm lifecycles, if you can suggest a fix, please do :)
@biancadanforth Just copying those files will not get you all of the utils. Could you try the npm link instructions and see if it works better for you? Thanks